packages/vx-text/package.json

```diff
@@ -29,11 +29,10 @@
   "homepage": "https://github.com/hshoff/vx#readme",
   "dependencies": {
     "@types/classnames": "^2.2.9",
-    "@types/lodash": "^4.14.146",
+    "@types/lodash.memoize": "^4.1.6",
     "@types/react": "*",
     "classnames": "^2.2.5",
-    "lodash": "^4.17.15",
-    "prop-types": "^15.7.2",
```

Comment:
we actually need this because we have a babel plugin that adds proptypes based on the types defined for a component.

this is useful for consumers I think (proptypes complain in dev / in the console), but @hshoff also was surprised by this recently so maybe we could consider removing it througout.

Reply:
ok!